### PR TITLE
homework 4 with massive

### PR DESCRIPTION
**Pull request by scononen**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_4/HomeWork4.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        System.out.println("количества нечетных элементов двухмерного массива ="+ Nechetnelement);
+
+        // Бонус
+//        Инициализация массива типа String и заполнение его именами коллег
+        String[] namearray = new String[]{"Nikita", "Alexei", "Ivan", "Timofei"};
+        for (int index = 0; index != namearray.length; index++) {
+
+            System.out.println(namearray[index]+" ");}
+
+        String [] Otherarray = new String[4];
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_4/HomeWork4.java]: not the best approach. either try String[] otherArray = new String[namearray.length] or String[] otherArray = namearray.clone()

File: src/main/java/org/example/homework_nr_4/HomeWork4.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            };
+// Вычесление суммы и среднего значения элементов одномерного массива
+        float sum = 0;
+        float count=0;
+        float avarage =0;
+        for (int j = 0; j != anotherarray.length; j++) {
+              sum += anotherarray[j];
+        }
+        for (int j = 0; j != anotherarray.length; j++) {
+            count++;
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_4/HomeWork4.java]: move that count into the loop above. no need to create one more loop when you can fit two instructions into one loop above

Comment (scononen) [reply src/main/java/org/example/homework_nr_4/HomeWork4.java]: was moved

File: src/main/java/org/example/homework_nr_4/HomeWork4.java (hunk 1 above)
Comment (scononen) [reply src/main/java/org/example/homework_nr_4/HomeWork4.java]: was changed to String[] otherArray = new String[namearray.length]

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**